src/modules/Dashboard/features/Participants/Participants.utils.tsx

```diff
@@ -1,26 +1,76 @@
 import { Dispatch, SetStateAction } from 'react';
 import { t } from 'i18next';
+import { format } from 'date-fns';
 
 import { Svg } from 'shared/components/Svg';
 import {
-  StyledSmallAppletImg,
-  StyledSmallAppletImgPlaceholder,
   StyledBodyMedium,
-  StyledLabelLarge,
   StyledFlexTopCenter,
+  StyledLabelLarge,
+  StyledSmallAppletImg,
+  StyledSmallAppletImgPlaceholder,
   variables,
 } from 'shared/styles';
 import { RespondentDetail, RespondentStatus } from 'modules/Dashboard/types';
 import { HeadCell } from 'shared/types';
 import i18n from 'i18n';
-import { MenuItemType } from 'shared/components';
+import { MenuItem, MenuItemType } from 'shared/components';
 import { checkIfCanAccessData, checkIfCanManageParticipants } from 'shared/utils';
+import { DateFormats } from 'shared/consts';
 
-import { ChosenAppletData, GetParticipantActionsProps } from './Participants.types';
+import {
+  ChosenAppletData,
+  GetParticipantActionsProps,
+  ParticipantActionProps,
+} from './Participants.types';
 import { ParticipantsColumnsWidth } from './Participants.const';
 
+/**
+ * Remove dividers that are not between normal displayed menu items
```

Comment:
Am I correct in this suggested description? It seems a bit clearer:
```suggestion
 * Reduce consecutive visible dividers to one
```

Reply:
It also hides dividers at the start and end of the list, so I think the existing description better summarises that